recipes/tippecanoe/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "tippecanoe" %}
+{% set version = "1.34.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/mapbox//{{name}}/archive/{{ version }}.tar.gz
+    sha256: 7a2dd2376a93d66a82c8253a46dbfcab3eaaaaca7bf503388167b9ee251bee54
+
+build:
+  skip: true   # [win]
+  number: 0
+  script:
+    - ls
+    - make -j
```

I think here it should be
```suggestion
    - make -j$CPU_COUNT
```